src/commands/build.ts

```diff
@@ -45,11 +45,12 @@ export class BuildCommand implements Command {
           'used for service worker generation.'
     },
     {
-      name: 'insert-dependency-links',
+      name: 'add-prefetch',
```

Comment:
I prefer the shorter `add-prefetch` when it comes to command line args. And `insert` always felt like it was inserting something new into my application while `add` feels like it's adding a new feature/ability to my application. So I prefer as-is, but happy to make the change if you feel strongly.